### ISSUE TITLE
It Fails to run and then cycles the plugin when two Amps are on the network

### ISSUE THREAD

**Issue by dazza120** [NorthernMan54/homebridge-yamaha-home#53]
Hi thanks fro the plugin its brilliant I've been using it for ages but now i have a problem. I have purchased another WXA-50 amp and when the two are on the network the plugin fails to start the bridge its a constant loops can you help please? Ive just X'd the IP here. Im using Hoobs 4 home bridge version 1.4.1 which is the latest for Hoobs I'm running on a PI3

17/07/2022, 08:43:47Yamaha Home Bridge starting
17/07/2022, 08:43:53Yamaha Home BridgeLoaded plugin 'homebridge-yamaha-home'
17/07/2022, 08:43:53Yamaha Home BridgeLoading 1 platforms...
17/07/2022, 08:43:53Yamaha Home Bridgeyamaha-homeGetting Yamaha AVR devices.
17/07/2022, 08:43:53Yamaha Home Bridgeyamaha-homePossible Yamaha device discovered Bath Room Amp [ 'XXX.XXX.X.XX' ]
17/07/2022, 08:43:53Yamaha Home Bridgeyamaha-homePossible Yamaha device discovered Dining Room Amp [ 'XXX.XXX.X.XX' ]
17/07/2022, 08:43:53Yamaha Home Bridgeyamaha-homeFound Yamaha WXA-50 - 0B3AE8B3, "Bath Room Amp"
17/07/2022, 08:43:53Yamaha Home Bridgeyamaha-homeChecking for 'inputs_as_accessories' for receiver 'Bath Room Amp'
17/07/2022, 08:43:53Yamaha Home Bridgeyamaha-homeFound Yamaha WXA-50 - 0DB591A3, "Dining Room Amp"
17/07/2022, 08:43:53Yamaha Home Bridgeyamaha-homeChecking for 'inputs_as_accessories' for receiver 'Dining Room Amp'
17/07/2022, 08:43:53Yamaha Home Bridgeyamaha-homeAdding zone controller for Main
17/07/2022, 08:43:53Yamaha Home Bridgeyamaha-homeAdding zone controller for Main
17/07/2022, 08:44:08Yamaha Home Bridgeyamaha-homeWaited 10 seconds, stopping discovery.
17/07/2022, 08:44:08Yamaha Home Bridgeyamaha-homeDiscovery finished, found 2 Yamaha AVR devices.
17/07/2022, 08:44:17Yamaha Home Bridge starting
17/07/2022, 08:44:22Yamaha Home BridgeLoaded plugin 'homebridge-yamaha-home'
17/07/2022, 08:44:23Yamaha Home BridgeLoading 1 platforms...
17/07/2022, 08:44:23Yamaha Home Bridgeyamaha-homeGetting Yamaha AVR devices.
17/07/2022, 08:44:23Yamaha Home Bridgeyamaha-homePossible Yamaha device discovered Bath Room Amp [ 'XXX.XXX.X.XX' ]
17/07/2022, 08:44:23Yamaha Home Bridgeyamaha-homePossible Yamaha device discovered Dining Room Amp [ 'XXX.XXX.X.XX' ]
17/07/2022, 08:44:23Yamaha Home Bridgeyamaha-homeFound Yamaha WXA-50 - 0B3AE8B3, "Bath Room Amp"
17/07/2022, 08:44:23Yamaha Home Bridgeyamaha-homeChecking for 'inputs_as_accessories' for receiver 'Bath Room Amp'
17/07/2022, 08:44:23Yamaha Home Bridgeyamaha-homeFound Yamaha WXA-50 - 0DB591A3, "Dining Room Amp"
17/07/2022, 08:44:23Yamaha Home Bridgeyamaha-homeChecking for 'inputs_as_accessories' for receiver 'Dining Room Amp'
17/07/2022, 08:44:23Yamaha Home Bridgeyamaha-homeAdding zone controller for Main
17/07/2022, 08:44:23Yamaha Home Bridgeyamaha-homeAdding zone controller for Main
17/07/2022, 08:44:38Yamaha Home Bridgeyamaha-homeWaited 10 seconds, stopping discovery.
17/07/2022, 08:44:38Yamaha Home Bridgeyamaha-homeDiscovery finished, found 2 Yamaha AVR devices.
17/07/2022, 08:44:47Yamaha Home Bridge starting

17/07/2022, 08:45:23Yamaha Home Bridgeyamaha-homeChecking for 'inputs_as_accessories' for receiver 'Bath Room Amp'
17/07/2022, 08:45:23Yamaha Home Bridgeyamaha-homeAdding zone controller for Main
17/07/2022, 08:45:23Yamaha Home Bridgeyamaha-homeAdding zone controller for Main

**Comment by dazza120** [comment]
It appears that the application can’t work with two amps, as it still seeks out other stuff and ignores IP address so that it would only look for the IP address entered

**Comment by marvelousmo** [comment]
I have the exact issue. Anything we can do about it?

**Comment by dazza120** [comment]
> I have the exact issue. Anything we can do about it?

Yeah use the Yamaha RXV one instead you get a few errors in the log but it works with multiple amps, would have loved the dev of Yamaha home to update it for multiple amps as I’ve used that for years but the dev doesn’t seem interested in doing that which is a shame as it was bulletproof.
